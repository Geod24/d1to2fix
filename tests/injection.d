void foo ( ) /* d1to2fix_inject: const */
{ }

void bar ( ) /* d1to2fix_inject: @nogc */
{ }
