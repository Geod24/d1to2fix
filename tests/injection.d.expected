void foo ( ) const
{ }

void bar ( ) <unsupported token injection>
{ }
